cmd/root.go

```diff
@@ -47,5 +47,8 @@ func Run() {
 	cmd.AddCommand(factory.Build(commands.Function))
 	cmd.AddCommand(factory.Build(commands.Schema))
 
+	// Uncomment once backend API and data models are complete
```

Comment:
[q] Is this something you'll do before merging this PR ?

Reply:
We will most likely come back and uncomment this once we've deployed the fully implemented data models/API in the back end. Technically this could be uncommented right now, but runs the risk of having functionality within the CLI (although hidden) that isn't yet available. @makesitgo feel free to chime in!

Reply:
I would either associate this to a ticket or create a small ticket for this, for TODOs that need to be addressed after the PR is merged we do:

`// TODO(REALMC-XXXX): do the thing`

Reply:
I imagine there's a point where you'll want this set of commands to be included in the CLI but things aren't yet fully released to prod?

if that's the case (maybe for testing purposes), you'll want to also mark this command as `Hidden`.  I would add that to the commented code here, just so we don't forget down the line.  Extra credit for filing the _other_ JIRA ticket to cover "un-hiding" the command and adding _that_ `TODO` as well...